pyproject.toml

```diff
@@ -10,5 +10,51 @@ version_scheme =  "post-release"
 local_scheme = "no-local-version"
 write_to =  "stravalib/_version_generated.py"
 
+[project]
+name = "stravalib"
+description = "A Python package that makes it easy to access and download data from the Strava V3 REST API."
+keywords = ["strava", "running", "cycling", "athletes"]
+readme = "README.md"
+dynamic =["version"]
+license = {text = "Apache 2.0 License"} 
+authors = [{name="Hans Lellelid", email="hans@xmpl.org"}]
+maintainers = [
+     {name = "Leah Wasser", email = "leah@pyopensci.org"},
+     {name = "Hans Lellelid", email = "hans@xmpl.org"},
+     {name = "Jonatan Smoocha"},
```

Comment:
Email is optional but if y'all want your email here or a different version of your name please feel free to add! @yihong0618 @jsamoocha

Reply:
ok there is indeed a pypi bug. confirmed. For this PR i'll remove emails. we can add them back when we have a working fix!